ci/jenkins/pipelines/skuba-conformance.Jenkinsfile

```diff
@@ -10,6 +10,23 @@ pipeline {
     }
 
     stages {
+
+        stage('Git Clone') { steps {
+            deleteDir()
+            checkout([$class: 'GitSCM',
+                      branches: [[name: "*/${BRANCH_NAME}"],[name: '*/master']],
+                      doGenerateSubmoduleConfigurations: false,
+                      extensions: [[$class: 'LocalBranch'],
+                                   [$class: 'WipeWorkspace'],
+                                   [$class: 'RelativeTargetDirectory', relativeTargetDir: 'skuba']],
+                      submoduleCfg: [],
+                      userRemoteConfigs: [[refspec: '+refs/pull/*/head:refs/remotes/origin/PR-*',
+                                           credentialsId: 'github-token',
+                                           url: 'https://github.com/SUSE/skuba']]])
+            dir("${WORKSPACE}/skuba") {
+                sh(script: "git checkout ${BRANCH_NAME}", label: "Checkout Branch")
```

Comment:
same here.  

Besides, I'm not sure if the  `branches` parameter is needed, given that we do the checkout manually.

Reply:
removed the manual checkout, use the BRANCH env instead so we directly pull into the proper branch/commit

Plus that allows us to use the changelog to get changes against master :o